Source/WebKit/Shared/WebCoreArgumentCoders.serialization.in

```diff
@@ -1476,9 +1476,18 @@ struct WebCore::WindowFeatures {
     [NotSerialized] Vector<String> additionalFeatures;
 };
 
-[Nested] enum class WebCore::CompositionUnderlineColor : bool
+header: <WebCore/CompositionAttribute.h>
 
-struct WebCore::CompositionUnderline {
+enum class WebCore::CompositionUnderlineColor : bool
+
+[CustomHeader] struct WebCore::CompositionHighlight {
+    unsigned startOffset;
+    unsigned endOffset;
```

Comment:
Could we either add a validator that end >= start, or have a start/length pair?  Otherwise we could deserialize an end less than a start and that could cause some issues.